Protein-gym.py

```diff
@@ -15,6 +15,9 @@
 from transformers import AutoTokenizer, DataCollatorWithPadding, AutoModelForSequenceClassification, TrainingArguments, Trainer
 from datasets import load_dataset
 from evaluate import load
+
+from protein_lm.tokenizer.tokenizer import EsmTokenizer, AptTokenizer
```

Comment:
You can do `from protein_lm.tokenizer import EsmTokenizer, AptTokenizer` here :)

Reply:
Hi, I just pushed the changes :)